README.md

```diff
@@ -0,0 +1,39 @@
+# Bookstore
+
+This app allows a user to add and store a new book, delete, edit and keep track of their reading progress.
+
+## Built With
+
+- React
+- CSS
+- Webpack
+
+## Live Demo
+
+Not yet added
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- In your terminal, move the location you want to clone the project.
+- Enter git clone git@github.com:kemigabocatherine/book-store.git
+- cd to the repo.
+- Open the project and use it as desired
+- **PS:** Use webpack to run and build project both locally and on the browser.
```

Comment:
This is a React project, I would recommend adding some steps on how users can use a react project locally (npm install, npm start ...) rather than asking them to use webpack to run and build the project.